sale_loyalty_discount_attribute/tests/test_reward_values_discount.py

```diff
@@ -0,0 +1,264 @@
+from odoo.fields import Command
+from odoo.tests import common
+
+
+class TestRewardValuesDiscount(common.TransactionCase):
+    def order_line_dict(self, product, qty=1):
+        return {
+            "name": product.name,
+            "product_id": product.id,
+            "product_uom_qty": qty,
+            "product_uom": product.uom_id.id,
+            "price_unit": 10.0,
+        }
+
+    def setUp(self):
```

Comment:
@diogocsc Could you change this to setUpClass ?

Reply:
Ping @diogocsc